content/en/03-using-pixie/03-api-quick-start.md

```diff
@@ -283,3 +283,9 @@ func (s *tableMux) AcceptTable(ctx context.Context, metadata types.TableMetadata
 	return &tablePrinter{}, nil
 }
 ```
+
+## End-to-End Encryption
+
+Pixie offers end-to-end encryption for telemetry data requested by the API. For more information, see the [FAQ](/about-pixie/faq#how-does-pixie-secure-its-data).
+
+Encryption is controlled by a flag during client creation. For additional info, see the [encryption example](https://github.com/pixie-labs/pixie/blob/main/src/api/go/pxapi/examples/encryption_example/example.go#L77) project.
```

Comment:
binary not project. Also should we link to a python example too?
https://github.com/pixie-labs/pixie/blob/main/src/api/python/examples/script_with_single_table.py#L35